frontend/components/Meetup/PresentationsList.vue

```diff
@@ -9,88 +9,96 @@ defineProps({
     type: Array,
     default: () => [],
   },
+  loading: {
+    type: Boolean, 
+    default: true,
+  },
 })
 
 </script>
 
 <template>
-  <div class="mt-12 overflow-hidden bg-white text-zinc-700">
-    <div class="border-b border-zinc-200 px-4 py-5 sm:px-6">
-      <h3 class="pl-4 text-xl font-medium leading-8">
+  <div v-if="!loading" class="mt-12 bg-white text-zinc-700 shadow-lg">
```

Comment:
Do we need that **loading** here?
We are already hiding this component in `Meetup` component.

```
<div v-if="!loading">
  <meetup-header :loading="loading" :meetup="meetup"/>
  <presentations-list :loading="loading" :presentations="meetup.presentations"/>
</div>
```

Reply:
You're right, I just commited a change for that